src/Evolution/Systems/Cce/System.hpp

```diff
@@ -114,12 +114,12 @@
  */
 namespace Cce {
 
-template <bool EvolvePartiallyFlatCartesianCoordinates>
+template <bool EvolveCcm>
```

Comment:
I'm a bit confused by this template parameter. The name suggests that in addition to CCE this would "evolve CCM" as well, but that doesn't make sense to me since the characteristic evolution is the same, right? Please add some documentation to this template parameter, and also refer to this documentation in other places where this template parameter occurs.

Reply:
Eventually the characteristic evolution will be able to do GH+CCM once we add those actions in. But we don't want to change any functionality of the stand alone CCE executable, so we add this template parameter to separate the two. Some docs explaining this could be nice

Reply:
That makes a lot of sense in the context of the executable, but an `EvolveCcm` template parameter on the CCE system confuses me.

Reply:
There's an additional evolved variable on the characteristic side for CCM

Reply:
You still need to add docs to this system. I'd just do

```
\note The template parameter `EvolveCcm` will add an extra evolved variable
to the characteristic system, namely `Cce::Tags::PartiallyFlatCartesianCoords`.
```

Reply:
> You still need to add docs to this system. I'd just do
> 
> ```
> \note The template parameter `EvolveCcm` will add an extra evolved variable
> to the characteristic system, namely `Cce::Tags::PartiallyFlatCartesianCoords`.
> ```

Alright, done! I also removed `Cce::ReducedWorldtubeDataManager` since it is outdated.